loadtest/main.go

```diff
@@ -195,42 +190,17 @@ func run(config Config) {
 		inactiveAccounts, activeAccounts = activeAccounts, inactiveAccounts
 	}
 
-	lastHeight := getLastHeight()
```

Comment:
We don't restrict number of txns per block now so this can be removed. Additionally, this was a tcp call so it could've caused some perf issues.